lib/refile/cloudinary.rb

```diff
@@ -24,7 +24,7 @@ class Cloudinary
     # @param [String] cloud_name The Cloudinary cloud name
     # @param [String] api_key The Cloudinary api key
     # @param [String] api_secret The Cloudinary api secret
-    def initialize(auth, max_size: 100)
+    def initialize(auth, max_size: 10_485_760)
```

Comment:
Perfect. As you have started working on tests, could you also add a test making sure that the default is of that value? Thanks!

Reply:
ok, I will do this.